cartridges/plugins/wso2am-1.9.0/wso2am-startup-handler.py

```diff
@@ -683,33 +719,54 @@ def get_host_name_from_cluster(self, service_name, app_id):
 
         return hostname
 
-    def read_member_ip_of_service(self, service_name, app_id):
+    def check_for_kubernetes_cluster(self, service_name, app_id):
         """
-        Get member ips from the topology for a service
-        :return: ip set
+        Check the deployment is kubernetes
+        :return: True
         """
-        clusters = None
-        topology = TopologyContext().get_topology()
-        members = None
-        member_ips = None
-
-        if topology.service_exists(service_name):
-            service = topology.get_service(service_name)
-            clusters = service.get_clusters()
-        else:
-            WSO2AMStartupHandler.log.error("[Service] %s is not available in topology" % service_name)
+        isKubernetes = False
+        clusters = self.get_clusters_from_topology(service_name)
 
         if clusters is not None:
             for cluster in clusters:
                 if cluster.app_id == app_id:
-                    members = cluster.get_members()
+                    isKubernetes = cluster.is_kubernetes_cluster
+
+        return isKubernetes
+
+
+    def get_clusters_from_topology(self, service_name):
+        """
+        get clusters from topology
+        :return: clusters
+        """
+        clusters = None
+        topology = TopologyContext().get_topology()
+
+        if topology is not None:
+            if topology.service_exists(service_name):
+                service = topology.get_service(service_name)
+                clusters = service.get_clusters()
+            else:
+                WSO2AMStartupHandler.log.error("[Service] %s is not available in topology" % service_name)
+
+        return clusters
 
-        if members is not None:
-            member_ips = ""
-            for member in members:
-                member_ips = member_ips + member.member_default_private_ip + ":"
-                WSO2AMStartupHandler.log.info("[Member_IPS] %s from topology" % member_ips)
 
-        return member_ips
+    def check_vm_or_kubernetes(self, use_kub, service_name, app_id):
+        """
+        Check for vm or kubernetes
+        :return: True for vm and False for kubernetes
+        """
+
+        if use_kub == 'false':
+            return True
```

Comment:
Why do we need this check inside this method? I do not think its appropriate to have this parameter use_kub.